app/client/src/PluginActionEditor/components/PluginActionForm/components/UQIEditor/hooks/useFormData.ts

```
@@ -0,0 +1,22 @@
+
+export const useFormData = () => {
+  const data = useSelector(getFormValues(QUERY_EDITOR_FORM_NAME)) as
+    | QueryAction
+    | SaaSAction;
+
+  const formEvaluation = useSelector(getFormEvaluationState);
+
+  let evaluationState = {};
+
+  // Fetching evaluations state only once the formData is populated
+  if (!!data) {
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)